src/mlpack/tests/conv_layer_test.cpp

```diff
@@ -0,0 +1,78 @@
+ * @author Nilay Jain
```

Comment:
Can you add ``/*`` so that we can automatically test the code? For more information here is the build log: https://travis-ci.org/mlpack/mlpack/builds/150478977